docs/content/direct/release-notes.md

```diff
@@ -6,12 +6,13 @@ The following sections list the known issues for each release. The issue list is
 
 The main advance in this release is finishing the implementation of the create-only feature. It is now available for use.
 
-### Remaining limitations in 0.24.0
+### Remaining limitations in 0.25.0 and its candidates
 
 * Although the create-only feature can be used with Job objects to avoid trouble with `.spec.selector`, requesting singleton reported state return will still lead to a controller fight over `.status.replicas` while the Job is in progress.
 * Removing of WorkStatus objects (in the transport namespace) is not supported and may not result in recreation of that object
 * Singleton status return: It is the user responsibility to make sure that if a BindingPolicy requesting singleton status return matches a given workload object then no other BindingPolicy matches the same object. Currently there is no enforcement of that.
 * Objects on two different WDSes shouldn't have the exact same identifier (same group, version, kind, name and namespace). Such a conflict is currently not identified.
+* The value of transport controller's `max-num-wrapped` flag is not handled properly, if the value leads to multiple ManifestWork objects with multiple workload objects in them.
```

Comment:
This remark belongs in the "remaining limitations" for some earlier releases too. This limitation has been with us since the commit that introduced this parameter. Following is the list of releases.

```console
mspreitz@mjs13 kubestellar % git tag --contains c72f175       
v0.24.0
v0.24.0-alpha.3-experiment.2
v0.24.0-alpha.3.experiment.1
v0.24.0-experiment.3
v0.24.0-experiment.4
v0.24.0-experiment.5
v0.24.0-rc.1
v0.25.0-alpha.1.test.1
v0.25.0-alpha.1.test.2
v0.25.0-rc.1
```

Reply:
Thanks! Updated in latest force-push.